### PR TITLE
Makefile glide experiment

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

**Comment by iamniting** [comment]
The good part is that it worked, But it got timeout after 3 hours. I think Centos CI has set a limit to run any CI for up to 3 hrs. They abort it after 3 hours. Can we request them to increase this time?

File: Makefile [hunk 1]
```diff
@@ -83,7 +83,7 @@ ifndef HGPATH
 endif
 	echo "Installing vendor directory"
 	if [ "$(GLIDE_HOME)" ]; then mkdir -p "$(GLIDE_HOME)"; fi
-	$(GLIDE) install -v
+	timeout 20m $(GLIDE) install -v || $(GLIDE) install -v
```
Comment (phlogistonjohn) [inline Makefile:86]: Use timeout -v and put the timeout on both glide invocations please.

**Review by phlogistonjohn — CHANGES_REQUESTED** [review]
Looks like the general gist of it worked. It just needs a few small tweaks and we can merge it.

File: Makefile (hunk 1 above)
Comment (iamniting) [reply Makefile:86]: I removed -v intentionally as I see that in centos timeout does not have that option

Screenshot from 2021-01-21 19-57-08

Comment (phlogistonjohn) [reply Makefile:86]: OK. We just want the 2nd invocation to also time out in case it gets stuck

**Review by phlogistonjohn — APPROVED** [review]

**Comment by iamniting** [comment]
> The good part is that it worked, But it got timeout after 3 hours. I think Centos CI has set a limit to run any CI for up to 3 hrs. They abort it after 3 hours. Can we request them to increase this time?

The PR has been raised to increase the timeout to 5 hrs https://github.com/gluster/centosci/pull/130

**Comment by iamniting** [comment]
The PR has passed now and it took less than 3 hrs after increasing the timeout.

Screenshot from 2021-01-22 16-44-00

**Comment by phlogistonjohn** [comment]
Fantastic. I will give @raghavendra-talur to the end of the day to look & review.

**Pull request merged**